releaseNotes/v7.1.4.md

```diff
@@ -20,14 +20,15 @@ Monday, Feb 11 2019
 - **Fixes CI to allow spaces in docker registry passwords**: CI jobs with [docker registry integration](http://docs.shippable.com/platform/integration/dockerRegistryLogin/) would fail if spaces were present in the docker registry password, as reported by [support/4655](https://github.com/Shippable/support/issues/4655). This bug has been fixed.
 - **Fixes issue with status not showing for some jobs**: Status for some jobs in subscription and project dashboard was not showing. This has now been fixed.
 - **Fixes issue with language versions specified as numbers are not working**: Specifying language versions as numbers resulted in CI builds running on default version instead of specified version. This is fixed now. Builds will now run on specified versions.
-- **Fixes loading the node pools page when user has no permission on running build**: Node Pools page was not loading for users that do not have permission to access the job that a node is currently running. This is now fixed. 
+- **Fixes loading the node pools page when user has no permission on running build**: Node Pools page was not loading for users that do not have permission to access the job that a node is currently running. This is now fixed.
 
 ## Shippable Server
 
 ### Features
 
 - **Build artifacts can be stored in Google Cloud Storage**: Shippable admin can now enable build artifacts like build consoles, coverage reports, and test reports to be stored on Google Cloud Storage file store. If both file stores (AWS S3 and Google Cloud Storage) are configured, admin can also map a file store provider to each node type (BYON, shared, on-demand).
 
+**Note:** To upgrade to v7.1.4 from a version below v6.12.4, first upgrade to v6.12.4 and then to v7.1.4.
```

Comment:
This doesn't make sense here. It needs to be tagged as a known issue in the release notes for v7.1.4 itself.